client/ssh.go

```diff
@@ -30,5 +32,18 @@ func SSHCLI(signer crypto.Signer, sshCert ssh.PublicKey, server string) ([]strin
 		return nil, err
 	}
 
-	return []string{"ssh", "-i", tmpdir, server}, nil
+	return []string{"ssh", "-i", tmpdir + "/id", server}, nil
+}
+
+func DefaultComment() string {
```

Comment:
I'm having a hard time deciding if this belongs in a `utils`, `cmd/hallow-cli/` or `client`. I think since this doesn't add dependencies, it's maybe fine here. I guess I'd be using this in my own custom client too.